app/models/UserAnswers.scala

```diff
@@ -51,6 +51,7 @@ final case class UserAnswers(
     }
   }
 
+  @deprecated
```

Comment:
deprecated is causing a warn on compile, to remove it we need to add a massage and date as a param (not important though)